### PR TITLE
Change Ticket button label and appearance when sale-status is 'unavai…

### PR DESCRIPTION
**Pull request by lemonboston**
…lable'. #221 

---

TODOs:
* backend: all events now seems to have ticket with sale-status "unavailable"
* finalize color and label for unavailable ticket button
* translations for the new label

### CODE REVIEW

**Comment by lemonboston** [comment]
Added the translations, made the dark theme ticket background darker, squashed and rebased.
Ready for review.

Please note that all events I've checked seem to have 'unavailable' status now, so I guess it makes sense to wait with merging until we see if it makes sense to use this.
(The branch `mx/245-simple-response-logging` is rebased on this one now, so you can use that to check the response logs while using the app.)

File: eventdiscovery-sdk/src/main/res/values/strings.xml
Comment (dmfs) [inline eventdiscovery-sdk/src/main/res/values/strings.xml:33]: Please add a comment that mentions the purpose of this string.

File: eventdiscovery-sdk/src/main/res/values/theme_dark.xml
Comment (dmfs) [inline eventdiscovery-sdk/src/main/res/values/theme_dark.xml:30]: The name is misleading. The button is not really disabled. It's just less prominent. Something like "reluctant" might be a better description. Alternatively we could try a regular non-colored button style (so the background of the butotn in the white theme is white as well). Maybe that's a better idea.

File: eventdiscovery-sdk/src/main/res/values/colors_light.xml
Comment (dmfs) [inline eventdiscovery-sdk/src/main/res/values/colors_light.xml:18]: Same here, the name is misleading.

File: eventdiscovery-sdk/src/main/res/values/colors.xml
Comment (dmfs) [inline eventdiscovery-sdk/src/main/res/values/colors.xml:7]: blank line?

File: eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/model/BasicTicketInfo.java
Comment (dmfs) [inline eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/model/BasicTicketInfo.java:48]: In cases like this I'm really annoyed by the design of Java's `Boolean` class. If it was an interface we wouldn't need this kind of class and method. We could just do

```
Boolean available = new TicketAvailability(ticketLink);
```

But that doesn't work unfortunately (probably because it wouldn't work with auto-unboxing).

However, how about something like this instead?
```
boolean available = new TicketAvailability(ticketLink).get();
```
with `TicketAvailability` being a `Lazy<Boolean>`?

**Review by dmfs — CHANGES_REQUESTED** [review]
done with the first round. We should indeed wait for the "unavailable" issue to be resolved before we merge this.

**Comment by dmfs** [comment]
Until the unavailable issue has been resolved you don't need to spend any time with this. We have other important things in the queue.

**Pull request closed without merging**
